website/views.py

```diff
@@ -98,11 +97,6 @@ def _render_nodes(nodes, auth=None, show_path=False):
 
 @collect_auth
 def index(auth):
```

Comment:
@pattisdr At the very least we don't need to `collect_auth` anymore on this view function.